### ISSUE TITLE
Update security levels screenshot

### ISSUE THREAD

**Issue by cstiens** [OnionBrowser/onionbrowser.github.io#22]
![OB_DEL-Security-Levels](https://user-images.githubusercontent.com/1434597/80062305-068f5000-84f9-11ea-813f-86d5040ddef2.png)

**Comment by m4mb01t4l14n0** [comment]
attached are images for dark and light mode new security levels, both from iPad
![security_levels_dm](https://user-images.githubusercontent.com/34868071/80098378-c51f9480-853a-11ea-971a-ac32509a5851.PNG)
![security_levels_lm](https://user-images.githubusercontent.com/34868071/80098379-c5b82b00-853a-11ea-89f3-89b4539f6841.PNG)

**Comment by tladesignz** [comment]
Aaaah, @m4mb01t4l14n0... these are the old ones! Please take the latest TestFlight to do screenshots!

**Comment by m4mb01t4l14n0** [comment]
better?
![security_levels_dm](https://user-images.githubusercontent.com/34868071/80102097-51808600-8540-11ea-9486-375d71891df7.PNG)
![security_levels_lm](https://user-images.githubusercontent.com/34868071/80102099-52191c80-8540-11ea-94c3-56936d5750d4.PNG)

**Comment by tladesignz** [comment]
Definitely! Sorry for the confusion! You might want to delete/edit your older post to stop the confusion spread.
